### PR TITLE
Models setup

### PR DESCRIPTION
**Pull request by tawnthanh**
I created all the models and migrations for us as well as the seeder files for states, random users, and random cities for our locations table. 

NOTE: I installed "faker"  to create fake users real quick. You might wonder why the locations seeder file doesn't utilize faker and it's because the address formatting is not consistent on the app. I wasn't sure how to isolate the state abbreviation, however, I have some code commented out in there. 

image

### CODE REVIEW

File: app/seeds/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from flask.cli import AppGroup
 from .users import seed_users, undo_users
-
+from .states import seed_states, undo_states
+from .locations import seed_cities
```
Comment (M-eLL) [inline app/seeds/__init__.py]: do we need to import undo_cities here? I saw that you created it in app/seeds/locations line 28

File: app/seeds/__init__.py [hunk 2]
```diff
@@ -9,10 +10,15 @@
...
+    seed_states()
+    seed_cities()
     # Add other seed functions here
 
 # Creates the `flask seed undo` command
 @seed_commands.command('undo')
 def undo():
     undo_users()
+    undo_states()
     # Add other undo functions here
```
Comment (M-eLL) [inline app/seeds/__init__.py:23]: `undo_cities()`?

File: app/models/survey.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+from .db import db
+
+# this is the through table
+answers = db.Table("answers", db.Model.metadata,
+    db.Column("requestId", db.Integer, db.ForeignKey("requests.id"), primary_key=True),
+    db.Column("questionId", db.Integer, db.ForeignKey("questions.id"), primary_key=True),
+    db.Column("answer", db.String, nullable=False))
```
Comment (M-eLL) [inline app/models/survey.py:7]: I think for the through table we only need the data for the two classes we are trying to tie together. So maybe we can get rid of line 7

**Review by M-eLL — CHANGES_REQUESTED** [review]
everything looks really great and I can tell you put a lot of hours into it! I put a few notes that I wasn't very confident about so maybe anna or arjun could offer another set of eyes. Thanks for doing all of that work Thanh!

File: app/models/survey.py (hunk 3 above)
Comment (tawnthanh) [reply app/models/survey.py:7]: line 7 is needed so we have a place to hold the answers between the request and question.

File: app/seeds/__init__.py (hunk 2 above)
Comment (tawnthanh) [reply app/seeds/__init__.py:23]: Will add that in!

**Pull request merged**